shapely/geometry/polygon.py

```diff
@@ -24,35 +24,39 @@ def _unpickle_linearring(wkb):
 
 class LinearRing(LineString):
     """
-    A closed one-dimensional feature comprising one or more line segments
+    A closed one-dimensional feature comprising one or more line segments.
```

Comment:
```suggestion
    A geometry type composed of one or more line segments that forms a closed loop (a line segment exists between the last and first coordinate).
```

Reply:
I left out the "(a line segment exists between the last and first coordinate).", because a LinearRing actually has a start and end coordinate pair that is identical (potentially automatically added, but once constructed, the closing coordinate pair is included in the representation), so it's not an implicit line segment between first and last coordinate pair.